src/OrbitGl/CaptureWindowTest.cpp

```diff
@@ -79,4 +81,193 @@ TEST_F(UnitTestCaptureWindow, SlidersRespondToMouseLeave) {
   EXPECT_FALSE(unit_test_vertical_slider->IsMouseOver());
 }
 
+class CaptureClientAppInterfaceFake : public orbit_capture_client::AppInterface {
+  [[nodiscard]] orbit_capture_client::CaptureClient::State GetCaptureState() const {
+    return orbit_capture_client::CaptureClient::State::kStopped;
+  }
+
+  [[nodiscard]] bool IsCapturing() const { return false; }
+
+  void StartCapture(){};
+  void StopCapture(){};
+  void AbortCapture(){};
+  void ToggleCapture(){};
+};
+
+class NavigationTestCaptureWindow : public CaptureWindow, public testing::Test {
```

Comment:
After looking at the details, I assume it's the latter, right? :sweat_smile:

Reply:
The latter. I probably could get around that for these tests, but looking a bit into the future, I assume that we want to extend this to cover more of the CaptureWindow, and this will require mocking some members.